GitExtensions/Program.cs

```diff
@@ -18,12 +18,20 @@ namespace GitExtensions
 {
     internal static class Program
     {
+        [System.Runtime.InteropServices.DllImport("user32.dll")]
+        private static extern bool SetProcessDPIAware();
+
         /// <summary>
         /// The main entry point for the application.
         /// </summary>
         [STAThread]
         private static void Main()
         {
+            if (Environment.OSVersion.Version.Major >= 6)
+            {
+                SetProcessDPIAware();
+            }
```

Comment:
For some reason, once I switched to the SDK-style project, the app would get rendered with a strange DPI awareness. I couldn't fix it with the app.manifest, but this call appears to resolve the issue.

Reply:
You could generate a binlog before/after and compare to see what's different about the project evaluations or targets being run.